applications/ContactStructuralMechanicsApplication/tests/ALM_frictional_contact_test_2D/hyper_simple_patch_test.mdpa

```diff
@@ -0,0 +1,108 @@
+Begin ModelPartData
```

Comment:
can't you do without mdpa if it is so simple?

Reply:
But this way is easier...